### ISSUE TITLE
<seal> element description should be expanded to other authentication mechanisms

### ISSUE THREAD

**Issue by jamescummings** [TEIC/TEI#1655]
The `<seal>` element http://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-seal.html is defined as: 

> contains a description of one seal or similar attachment applied to a manuscript.

Since seals are not just a physical aspect of the document, but are used as an authentication and/or authorisation mechanism the description here should allow for other things which serve this function but are not a 'similar attachment' to the document. e.g. stamps, authorising signature, stamped name blocks, chirograph documents, etc.  It would seem that `<seal>` might be an appropriate place to discuss the physicality of these authentication mechanisms regardless of whether they are something 'attached' or stamped, signed, or otherwise created.

**Comment by sydb** [comment]
I think @jamescummings has a point, but I think it is really 10.7.3.2 that would need to be updated, as much or more than the definition itself. And the description would have to reinforce that this is part of the _physical_ description of the document. But I typically defer to those who know more about manuscripts than I on this sort of thing. (E.g., David Birnbaum and Matthew Driscoll)

**Comment by jamescummings** [comment]
@sydb Yes, the prose there is really what should be updated, but I was just thinking a mild change to the description such as 'of one seal or similar attachment, stamps or other authentication mechanisms'.  It is that it needs to be physically 'attached' rather than say 'stamped' or 'embossed' that I'm objecting to really.  While all things are attached in one way or another this seems to indicate a particular type of seal. 

(I might also casually point out that we should remove references to 'manuscript' in such elements except where absolutely necessary since we also recommend the msDesc element for early printed books, etc. but that is only a side note.)

**Comment by sydb** [comment]
I agree @jamescummings .

**Comment by jamescummings** [comment]
Proposal: 
Seal current description: 

> **contains a description of one seal or similar attachment applied to a manuscript**

Proposed rewording:

> **contains a description of one seal, stamp, endorsement, or other authentication mechanism applied to the object described**

Any thumbs down?

**Comment by RobertoRDT** [comment]
Makes sense to me.

**Comment by chr-emil** [comment]
A vax seal connected to a document by a strip of parchment is very different from a stamp. The parchment strip may also have text written on it. One should focus on function or physical structure, I think.

We suggested some years ago (2013)  that one should move all the attributes and  elements describing the physical features from  msDesc  to ObjectDesc. Maybe this  work should be continued?

**Comment by holfordm** [comment]
I'm not sure that the definition of `<seal`> should become so loose. I would favour adopting what the charter encoding initative (CEI) has done with the element `<authDesc>`  http://www.cei.lmu.de/element.php?ID=15 which "describes the type of authentification used on the document" and can contain the CEI elements `<sealDesc>, <subscriptio>, <chirograph>` . `<authDesc>` certainly seems like what James is describing and we might consider adopting the other elements as well perhaps.

**Comment by holfordm** [comment]
No, probably not. My impression is that it's relatively unusual for documents to be desribed in TEI if CEI would be more appropriate.
But I do think that there are limits to how far the meaning of seal should be pushed. `<seal type="chirograph">`, `<seal type="endorsement">`, for example, would look wrong, even absurd to me as a medieval historian. 
I think it would be better to introduce a more general element (such as <authDesc>) for physical authentication mechanisms, which would belong in `<physDesc>` and would contain for the moment `<sealDesc>` or descriptive text + attributes for other types. e.g. `<authDesc type="chirograph">Chirograph</authDesc>`
However (separately) I do support the change of wording within the description of `<seal>` since there are different methods of sealing, and seals may certainly be applied as well as attached.

**Comment by jamescummings** [comment]
Since `<seal>` is part of `<sealDesc>` which is in `<physDesc>` it does make sense that any modification of the use of `<seal>` is still related to its physicality.  I think @holfordm is right that there is a point at which it isn't related to seals any more but things like chirographs. The nature of the authentication mechanisms of a document (whether by seal, chirograph, endorsement, or similar) should be treated separately from the physicality of the seals, stamps, embossments, or similar. 

I would still support loosening `<seal>` slightly to be non-specific to manuscript and to resist that the seal needs to be a typical wax seal attached to the manuscript, but could be an ink stamp, an embossing, or other such things of which you want to describe the physical appearance and details. 

I would separately also support importing some of the CEI elements in this area but that is a separate discussion I think.

@chr-emil noted that we had previously attempted to get some people to create a generic `<object>` (since `<objectDesc>` already exists) element of which `<msDesc>` would then be a specialisation for manuscripty thing. This would enable the description of non-manuscript non-incunabula objects that optionally are text-bearing.  (Let's say the sword excalibur as an example.)  The attempts to do that seem to have floundered and perhaps should be rebooted. Indeed the lingering hope that we might re-use some elements there is part of the reason why I prefer to be non-specific when descriptions say things like "to a manuscript", but also since we already suggest msDesc for non-manuscript materials.

**Revised Proposal:** 

Seal current description:
> **contains a description of one seal or similar attachment applied to a manuscript**

Proposed rewording:
> **contains a description of one seal, stamp, embossment, or similar applied to the object described**

sealDesc current description:
> ** describes the seals or other external items attached to a manuscript, either as a series of paragraphs or as a series of distinct seal elements, possibly with additional decoNotes.**

Proposed rewording:
> ** describes the seals, stamps, embossments or similar related to the object described either as a series of paragraphs or as a series of seal elements**


(I would remove the bit about decoNotes, because while it is true, you can also use `<condition>` etc. so I don't think this should be in the definition. That is better put in notes/remarks concerning the element.)

**Comment by lb42** [comment]
Coincidentally, I was thinking about the  &lt;object> element yesterday: there is a skeleton ODD and some discussion which SPQR was working on back in 2005  (see https://github.com/TEIC/TEI/issues/327 and https://github.com/TEIC/TEI/tree/dev/Documents/Object ) I was looking in response to a question from some Fr art historians wishing to record information about specific buildings (e.g. the Pantheon in Paris) and even particular works of art (say, the Mona Lisa). I'd be pleased to help revive/push this along a bit if there's a broad consensus in favour.

**Comment by jamescummings** [comment]
@lb42 I'm certainly in favour of getting something like this and would be glad to help.

**Comment by holfordm** [comment]
I'm sure the epigraphers for one would be interested!

**Comment by GVogeler** [comment]
I'm a bit late to comment, but @holfordm has fortunately pointed you already to the CEI. I would strongly support to keep information on authentication separate from information on physical feature of a manuscript that can (but don't have to) be used for authentication. Authentication can be done by text (signatures), by symbols (diplomatics is full of examples, don't hesitate to ask, if you need them, this might include stamps, but thinking of Chinese emperors or Japanese 'signature' they can be text as well), by objects attached (seals), or code (public keys, hash sums etc.). Having a "authentication" structure also in the teiHeader (e.g.) would allow adding means of authentication even to born digital texts.
So it's fine to keep authentication out of the scope of physical features/objectDesc.

Another issue is mixing seals with other embossed traces on the manuscript. This is at least in the line of what people in the past did. But why not distinguishing? We have `rs`, `name`, `persName`. So why not have `embossement` (or whatever native English is for this) as the more abstract element for `seal` and `stamp`? `seal` can carry futher object descriptions (material, dimensions, shape, modes of affixture ..., maybe @chr-emil has already the necessary generic model for objects to throw it in here), which `stamp` dos not have.

**Comment by chr-emil** [comment]
?We could make this an issue for the TEI meeting in November.  A signature is an authentication similar to a vax seal. So then we would need to include signature in the same overarching element as one should put stamp, vax and other  seals.


Best

Christian-Emil

________________________________

**Comment by hcayless** [comment]
Council subgroup thinks James should do the proposed rewording. If there's a need for a new structure to describe document authentication then a new issue should be opened with a proposal.

**Comment by lb42** [comment]
The word "stamp" is ambiguous in English. It could mean the impression of something like a signet, and hence is analogous to a seal. It can mean a separate little printed document such as a postage stamp. The two may or may not be functionally equivalent.

**Comment by jamescummings** [comment]
Here is a revised proposal for the simple rewording of descs without any schema changes. I think this are uncontroversial so will go ahead an implement them, but shout now if you see anything you don't like.

**Revised Proposal:** 

Seal current description:
> **contains a description of one seal or similar attachment applied to a manuscript**

Proposed rewording:
> **contains a description of one seal or similar applied to the object described**

sealDesc current description:
> ** describes the seals or other external items attached to a manuscript, either as a series of paragraphs or as a series of distinct seal elements, possibly with additional decoNotes.**

Proposed rewording:
> ** describes the seals or similar items related to the object described either as a series of paragraphs or as a series of seal elements**

**Comment by martinascholger** [comment]
@jamescummings : I have noticed that the proposed rewording has been done. Can we close the issue?
